plugin/trino-snowflake/src/test/java/io/trino/plugin/snowflake/TestSnowflakeConnectorTest.java

```diff
@@ -0,0 +1,351 @@
+    }
+
+    @Test
+    public void testViews()
+    {
+        onRemoteDatabase().execute("CREATE OR REPLACE VIEW tpch.test_view AS SELECT * FROM tpch.orders");
+        assertQuery("SELECT orderkey FROM test_view", "SELECT orderkey FROM orders");
+        onRemoteDatabase().execute("DROP VIEW IF EXISTS tpch.test_view");
+    }
+
+    // Override this test because the type of row "shippriority" should be bigint rather than integer for snowflake case
+    @Test
+    @Override
+    public void testDescribeTable()
+    {
+        MaterializedResult expectedColumns = resultBuilder(getSession(), VARCHAR, VARCHAR, VARCHAR, VARCHAR)
+                .row("orderkey", "bigint", "", "")
+                .row("custkey", "bigint", "", "")
+                .row("orderstatus", "varchar(1)", "", "")
+                .row("totalprice", "double", "", "")
+                .row("orderdate", "date", "", "")
+                .row("orderpriority", "varchar(15)", "", "")
+                .row("clerk", "varchar(15)", "", "")
+                .row("shippriority", "bigint", "", "")
+                .row("comment", "varchar(79)", "", "")
+                .build();
+        MaterializedResult actualColumns = computeActual("DESCRIBE orders");
+        assertEquals(actualColumns, expectedColumns);
+    }
+
+    // Override this test because the type of row "shippriority" should be bigint rather than integer for snowflake case
+    @Test
+    @Override
+    public void testShowCreateTable()
+    {
+        assertThat(computeActual("SHOW CREATE TABLE orders").getOnlyValue())
+                .isEqualTo("CREATE TABLE snowflake.tpch.orders (\n" +
+                        "   orderkey bigint,\n" +
+                        "   custkey bigint,\n" +
+                        "   orderstatus varchar(1),\n" +
+                        "   totalprice double,\n" +
+                        "   orderdate date,\n" +
+                        "   orderpriority varchar(15),\n" +
+                        "   clerk varchar(15),\n" +
+                        "   shippriority bigint,\n" +
+                        "   comment varchar(79)\n" +
+                        ")\n" +
+                        "COMMENT ''");
+    }
+
+    // Override this test because the origin test has an error of "This connector does not support setting column comments"
+    // So I override it by using execute correct SQL.
+    @Test
+    @Override
+    public void testCommentColumn()
+    {
+        String tableName = "test_column_comment_" + randomTableSuffix();
+        onRemoteDatabase().execute("CREATE TABLE tpch." + tableName + " (col1 bigint COMMENT 'test comment', col2 bigint COMMENT '', col3 bigint)");
+
+        assertQuery(
+                "SELECT column_name, comment FROM information_schema.columns WHERE table_schema = 'tpch' AND table_name = '" + tableName + "'",
+                "VALUES ('col1', 'test comment'), ('col2', null), ('col3', null)");
+
+        assertUpdate("DROP TABLE " + tableName);
+    }
```

Comment:
Set `SUPPORTS_COMMENT_ON_COLUMN` as false in `hasBehavior` and remove this override. 

Please check all subsequent test methods.